.clang-format

```diff
@@ -40,12 +40,12 @@ ConstructorInitializerAllOnOneLineOrOnePerLine: true
 DerivePointerBinding: false
 PointerBindsToType: true
 
-PenaltyExcessCharacter: 1000
-PenaltyBreakBeforeFirstCallParameter: 19
-PenaltyBreakComment: 60
-PenaltyBreakFirstLessLess: 1000
+PenaltyExcessCharacter: 50
+PenaltyBreakBeforeFirstCallParameter: 30
+PenaltyBreakComment: 1000
+PenaltyBreakFirstLessLess: 10
```

Comment:
Interesting, what is the actual outcome of these changes? I didn't see too many changes with the STREAM macros for instance.

Reply:
There are some examples, where a longer line is accepted, e.g. https://github.com/ros-planning/moveit/pull/2177/commits/e6a9b22035c607238dadf780c02120f32b6ba310#diff-3e42fc5ecfcb692fcc35b27df4d33675R171